packages/common/src/ui/components/domain/Dashboard/StatsPanel.tsx

```diff
@@ -30,7 +30,7 @@ export const StatsPanel: FC<StatsPanelProps> = ({
           color: 'gray.main',
           fontSize: '12px',
           fontWeight: 500,
-          marginLeft: '8px',
+          marginInlineStart: '8px',
```

Comment:
`margin-inline-start` and `margin-inline-end` are new to me!
Not supported by IE 11 or some older browsers, but much neater to use than checking with `useRTL` everywhere! and the intention is clearer, so I think a good habit to use it.